app/views/errors.py

```diff
@@ -95,14 +96,21 @@ def get_tx_id():
     return tx_id
 
 
+def get_account_url():
+    account_url = current_app.config['RESPONDENT_ACCOUNT_URL']
+    if ACCOUNT_URL in cookie_session:
```

Comment:
If `cookie_session[ACCOUNT_URL` is present but null then `None` will be rendered in the `href`. We should check that there is a non-empty value for the URL.